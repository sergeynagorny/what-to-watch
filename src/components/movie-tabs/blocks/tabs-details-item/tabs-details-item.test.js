import React from "react";
import renderer from "react-test-renderer";
import TabsDetailsItem from "./tabs-details-item.jsx";


it(`Should TabsDetailsItem render correctly`, () => {
  const tree = renderer.create(
      <TabsDetailsItem
        name={`name`}
      >
        {`children`}
      </TabsDetailsItem>
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
