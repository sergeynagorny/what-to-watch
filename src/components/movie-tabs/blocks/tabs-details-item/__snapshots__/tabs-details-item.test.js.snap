// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should TabsDetailsItem render correctly 1`] = `
<p
  className="movie-card__details-item"
>
  <strong
    className="movie-card__details-name"
  >
    name
  </strong>
  <span
    className="movie-card__details-value"
    style={
      Object {
        "whiteSpace": "pre-line",
      }
    }
  >
    children
  </span>
</p>
`;
